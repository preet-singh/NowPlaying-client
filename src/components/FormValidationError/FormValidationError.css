.error-message {
  color: orangered;
  font-weight: bold;
}