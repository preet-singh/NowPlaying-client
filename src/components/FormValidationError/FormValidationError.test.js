import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import FormValidationError from './FormValidationError';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

describe('Form validation error testing', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <FormValidationError />
            </BrowserRouter>,
            div
        );
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the error message', () => {
        const wrapper = shallow(<FormValidationError />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });

});