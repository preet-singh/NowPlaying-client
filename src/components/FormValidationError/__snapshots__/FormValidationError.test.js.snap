// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form validation error testing renders the error message 1`] = `<Fragment />`;
